client/src/components/CategoryView/CategoryView.js

```diff
@@ -4,10 +4,20 @@ import { Button, Col, Divider, Row, Typography } from "antd";
 
 import PlaceList from "../PlaceList/PlaceList";
 import { PlusOutlined } from "@ant-design/icons";
+import { useSelector } from "react-redux";
 
 function CategoryView() {
   const { Title } = Typography;
-  let categoryType = "Restaurants 🍝";
+  const categories = useSelector((state) => state.categories.allCategories);
+  const currentCategoryID = useSelector(
+    (state) => state.categories.currentCategoryID
+  );
+  let currentCategory = categories.find(
+    (category) => category.category_id === currentCategoryID
+  );
+  let categoryType = currentCategory.name + " " + currentCategory.emoji;
+  let btnCategory = currentCategoryID === 1 ? "Beach" : "Restaurant";
```

Comment:
This looks very hard coded. We will be adding other categories later.

Reply:
@sassansh yes we have to think of something later.